src/Stripe/Infrastructure/Mapper.cs

```diff
@@ -6,6 +6,18 @@ namespace Stripe
 {
     public static class Mapper<T>
     {
+        public static StripeList<T> MapStripeListFromJson(string json, string token = "data")
```

Comment:
This is very cool. :) I will do some simplification for the two methods to share code, but this is a great idea - thanks!